force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -214,10 +272,19 @@ public with sharing class ClassesObjectsMethods {
     public static Integer calculateYearDifference(Date d1, Date d2) {
 
         // Subtract the difference in years
-
-        return null; // Replace null with the variable you used to store the result
+        Integer d1Year = 0;
+        Integer d2Year = 0;
+
+        if(d1 == null || d2 == null){
```

Comment:
Typically add spaces around the parentheses 
 `if (d1 == null || d2 == null) {`